tests/Doctrine/Tests/DBAL/Platforms/AbstractPlatformTestCase.php

```diff
@@ -563,6 +564,22 @@ public function testGetDefaultValueDeclarationSQLForIntegerTypes()
         }
     }
 
+    public function testGetDefaultValueDeclarationSQLForDateType()
+    {
+        $currentDateSql = $this->_platform->getCurrentDateSQL();
+	    foreach (array('date', 'date_immutable') as $type) {
+		    $field = array(
+			    'type'    => Type::getType($type),
+			    'default' => $currentDateSql,
+		    );
+		
+		    $this->assertEquals(
```

Comment:
Please use `self::assert*` instead, and it would be better to use `assertSame()` in this case